src/content/getting-started/architecture/globalnet/_index.en.md

```diff
@@ -97,6 +97,21 @@ replying to DNS queries.
 
 Nothing extra needs to be done to build `submariner-globalnet` as it is built with the standard Submariner build.
 
+## Prerequisites
+
+Allow Globalnet controller to create/update/delete the Service with externalIPs field by below ways:
```

Comment:
@sridhargaddam Thank you for your feedback. Added this as a note to clarify on why (As @dfarrell07 pointed out [here](https://github.com/submariner-io/submariner-website/pull/693#discussion_r822882821), it would be good to separate what and why). Also, I've changed to numbered list to show that users need to both of the steps.